core/src/main/java/io/aiven/klaw/service/TopicControllerService.java

```diff
@@ -1348,6 +1344,16 @@ public List<TopicRequest> getExistingTopicRequests(
             tenantId);
   }
 
+  public boolean existsTopicRequests(TopicRequestModel topicRequestModel, int tenantId) {
```

Comment:
This method has been added but it hasn't been used in line 303 or 226

Reply:
I see it is being used in the next class for validation. Pls ignore comment I can see why it wasnt used in the previous lines.